test/algconvergence/ode_convergence_tests.jl

```diff
@@ -33,6 +33,9 @@ testTol = 0.2
   sim3 = test_convergence(dts2,prob,KuttaPRK2p5(threading=false))
   @test sim3.𝒪est[:l∞] ≈ 5 atol=testTol
 
+  sim3 = test_convergence(dts2,prob,RKO65())
+  @test sim3.𝒪est[:l∞] ≈ 5 atol=testTol
```

Comment:
what result are you getting here?

Reply:
Expression: ≈(sim3.𝒪est[:l∞], 5, atol=testTol)
   Evaluated: 0.9645038959802423 ≈ 5 (atol=0.2)

Expression: ≈(sim3.𝒪est[:l∞], 5, atol=testTol)
   Evaluated: 0.9645038959807267 ≈ 5 (atol=0.2)

Reply:
With the updated value I get: 

Expression: ≈(sim3.𝒪est[:l∞], 5, atol=testTol)
   Evaluated: 4.9872356619789535 ≈ 5 (atol=0.2)


Expression: ≈(sim3.𝒪est[:l∞], 5, atol=testTol)
   Evaluated: 4.998263889697244 ≈ 5 (atol=0.2)